devel/sccs/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                sccs
+version             5.09
+categories          devel
+platforms           darwin
+supported_archs     i386 x86_64
+license             CDDL
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+conflicts           cssc
+
+description         SCCS version control system
+
+long_description    The UNIX Source Code Control System
+
+homepage            http://sccs.sourceforge.net
+
+master_sites        sourceforge
+extract.suffix      .tar.bz2
```

Comment:
This line should change to `use_bzip2 yes`, otherwise MacPorts will try to extract it as a gzip file, which will work on OS X 10.9 and later due to enhancements Apple made to the tar command, but will fail on 10.8 and earlier.

Reply:
When I wrote that I felt that it wasn't quite right. Thanks, updated.